### PR TITLE
HPCC-21824 Added ES Translations

### PR DESCRIPTION
**Pull request by kunalaswani**
Added Spanish Translations to 7.2.x

Signed-off-by: Aswani, Kunal <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [x] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@rpastrana please review

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ define(
     Compress: "Comprimir",
     Compressed: "Comprimido",
     CompressedFileSize: "Tamaño de Archivo Comprimido",
+    ComputerUpTime: "Tiempo de trabajo de computadora ",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: is the extra space at the end needed?

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 2]
```diff
@@ -113,9 +114,12 @@ define(
     Contents: "Contenidos",
     ContentType: "Tipo de Contenido",
     ContinueWorking: "Sigue trabajando",
+    Copied: "Copiado ",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: extra space

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 3]
```diff
@@ -519,6 +531,7 @@ define(
     Preflight: "Pre-volado",
     PreloadAllPackages: "Precargar todos los paquetes",
     PreserveCompression: "Preservar Compression",
+    PressCtrlCToCopy: "Oprime ctrl+c para copiar",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: Oprima

**Review by rpastrana — COMMENTED** [review]
@kunalaswani @miguelvazq just a couple of comments

**Comment by kunalaswani** [comment]
Changes made, @rpastrana

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 4]
```diff
@@ -113,9 +114,12 @@ define(
     Contents: "Contenidos",
     ContentType: "Tipo de Contenido",
     ContinueWorking: "Sigue trabajando",
+    Copied: "Copiado",
     Copy: "Copia",
     CopyToClipboard: "Copiar al portapapeles",
+    CopyWUIDs: "Copiar WUIDs al portapapeles",
     Count: "Número",
+    CPULoad: "Carga de CPU ",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: extra space

**Review by rpastrana — APPROVED** [review]
@kunalaswani just one extra space left

**Comment by kunalaswani** [comment]
All spaces removed.

**Comment by kunalaswani** [comment]
@rpastrana can you re-review, all spaces removed.

**Review by rpastrana — APPROVED** [review]
@kunalaswani approved

**Comment by richardkchapman** [comment]
Why was this targetting master?

**Pull request merged**